TidepoolOnboarding/Views/Sections/YourDevicesViews.swift

```diff
@@ -56,8 +57,15 @@ fileprivate struct YourDevicesNotificationsView: View {
                 LocalizedString("Enable Critical Alerts in your iPhone or iPod Touch Settings", comment: "Onboarding, Your Devices section, Notifications view, list 2, item 1")
             )
             .startingAt(2)
-            .padding(.vertical)
+            .padding(.top)
             Paragraph(LocalizedString("Critical Alerts may be configured to alert you to higher risk situations while using Tidepool Loop, such as urgent low glucose, insulin pump occlusions, or other serious system errors.", comment: "Onboarding, Your Devices section, Notifications view, paragraph 3"))
+            .padding(.vertical)
```

Comment:
[nit] Please indent.

Reply:
I'll try to follow the whitespace/indent pattern in the rest of the file.

Reply:
Ah, yes, you pulled the closing paren for the `Paragraph` down to the next line. The rest of the code has it on the same line and then the `.padding(...)` would be indented automatically.